govuk-public-api-rules/ruleset.yaml

```
@@ -0,0 +1,2 @@
+extends:
```

I think in the other repo we decided to prefer this layout to avoid introducing extra whitespace changes in the future

```suggestion
extends:
  - "spectral:oas"
  - all
```

---

Good thinking - this is actually an odd case, where we've got a list inside a list, so I made the choice that this was a better version than:

```yaml
extends:
  -  - spectral:oas
     - all
```